bucket/rtlutility.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.0.3",
+    "description": "Tool for measuring the Round Trip Latency of your Digital Audio Workstation (DAW) and audio interface",
+    "homepage": "https://oblique-audio.com/rtl-utility.php",
+    "license": "Freeware",
+    "suggest": {
+        "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_x64.exe#/RTL_Utility.exe",
+            "hash": "5764c78d4e7150d187dfd9312b21a77c4d4cbecf54586f83cea7f83ae6918a2b"
+        },
+        "32bit": {
+            "url": "https://oblique-audio.com/downloads/RTL_Utility_1_0_3_Win32.exe#/RTL_Utility.exe",
+            "hash": "c788a1792e7d89c498268abaa9e6663bbbabfa696edc6f7ccdfdd862d00c786d"
+        }
+    },
+    "shortcuts": [
+        [
+            "RTL_Utility.exe",
+            "RTL Utility"
+        ]
+    ],
+    "checkver": {
+        "url": "https://oblique-audio.com/rtl-utility.php",
+        "re": "RTL_Utility_(?<version>(\\d+)_(\\d+)_(\\d+))_x64\\.exe",
+        "replace": "${1}.${2}.${3}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://oblique-audio.com/downloads/RTL_Utility_$matchVersion_x64.exe#/RTL_Utility.exe"
```

Comment:
```suggestion
                "url": "https://oblique-audio.com/downloads/RTL_Utility_$underscoreVersion_x64.exe#/RTL_Utility.exe"
```

Reply:
Ahh, totally missed this on  https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#version-variables